### PR TITLE
Date changed to Jan 12, 2021

### PR DESCRIPTION
**Pull request by Code-Aditya-14**

### CODE REVIEW

**Comment by Code-Aditya-14** [comment]
image

File: connect.php [hunk 1]
```diff
@@ -3,7 +3,11 @@
 $host = 'localhost';
 $db   = 'infinitoAdmin';
 $user = 'root';
```
Comment (debarghya472) [inline connect.php]: 1. Delete the overheads! "<<<< HEAD ..."
2. Watch the videos to setup db 
3. Share screenshots of the working website on localhost

**Review by debarghya472 — CHANGES_REQUESTED** [review]
See the review and do the changes accordingly

File: connect.php [hunk 2]
```diff
@@ -1,10 +1,9 @@
 <?php
-include './config.php';
 date_default_timezone_set('Asia/Calcutta');
 $host = 'localhost';
 $db   = 'infinitoAdmin';
 $user = 'root';
-$pass = DBPASSWORD;
```
Comment (debarghya472) [inline connect.php]: @Code-Aditya-14 Do not hardcore $pass as sql password will be different for everyone. So instead create a config.php file with DBPASSWORD as key to store sql password. 

Follow the docs in [contributing.md](https://github.com/Infinito-IIT-Patna/Infinito2020/blob/master/CONTRIBUTING.md)

**Review by debarghya472 — COMMENTED** [review]
Also when ur **requested for changes**  after u commit u need to inform to **re-review**. Otherwise we won't be notified.

**Comment by Code-Aditya-14** [comment]
image

File: connect.php [hunk 3]
```diff
@@ -1,10 +1,9 @@
 <?php
```
Comment (debarghya472) [inline connect.php]: Take account of the following points
1. do not remove include statement.
2. DBPASSWORD shouldn't be within single quotes ''

**Review by debarghya472 — CHANGES_REQUESTED** [review]
Good work! Just a new changes left and then ur good to go! If u have any doubts dm me

**Review by debarghya472 — APPROVED** [review]
good job!11

**Pull request merged**